### PR TITLE
repositories: add opentransactions

### PR DESCRIPTION
**Pull request by justusranvier**

### CODE REVIEW

**Comment by thesamesam** [comment]
Could you add `Signed-off-by` please? Thanks.

**Comment by justusranvier** [comment]
I updated the commit

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -2881,6 +2881,18 @@
     <source type="git">git+ssh://git@github.com/jcaesar/openclonk-overlay.git</source>
     <feed>https://github.com/jcaesar/openclonk-overlay/commits/master.atom</feed>
   </repo>
+  <repo quality="stable" status="official">
+    <name>opentransactions</name>
+    <description lang="en">Open-Transactions Gentoo Overlay</description>
+    <homepage>https://github.com/Open-Transactions/gentoo-overlay</homepage>
```
Comment (thesamesam) [inline files/overlays/repositories.xml:2887]: hm, am I missing something? https://github.com/Open-Transactions/gentoo-overlay doesn't exist..

Comment (justusranvier) [reply files/overlays/repositories.xml:2887]: It was accidentally created as private. The visibility is fixed now.

**Pull request closed without merging**